ansible/configs/tech-networking-901/setup_lab_user.yml

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
we can used control-user role instead of this task file

Reply:
should I also remove bastion-student-user role?

Reply:
leave it we can do it next iteration